daemons/controld/controld_fsa.c

```diff
@@ -240,11 +239,13 @@ s_crmd_fsa(enum crmd_fsa_cause cause)
         fsa_data = NULL;
     }
 
-    if ((fsa_message_queue != NULL) || (fsa_actions != A_NOTHING)
+    if ((fsa_message_queue != NULL)
+        || (controld_globals.fsa_actions != A_NOTHING)
         || pcmk_is_set(controld_globals.flags, controld_fsa_is_stalled)) {
-        crm_debug("Exiting the FSA: queue=%d, fsa_actions=%#llx, stalled=%s",
+        crm_debug("Exiting the FSA: queue=%d, "
+                  "controld_globals.fsa_actions=%#llx, stalled=%s",
```

Comment:
no need in log

Reply:
Drop the whole `crm_debug()` call or `fsa_actions` or what? I don't understand what we'd want to change here and why; seems reasonable for debugging or at least tracing

Reply:
Oh! Yeah, that makes sense.